docs/rules/wrap-iife.md

```diff
@@ -18,12 +18,18 @@ This rule requires all immediately-invoked function expressions to be wrapped in
 
 ## Options
 
-The rule takes one option which can enforce a consistent wrapping style:
+This rule has two options, a string option and an object option.
+
+String option:
 
 * `"outside"` enforces always wrapping the *call* expression. The default is `"outside"`.
 * `"inside"` enforces always wrapping the *function* expression.
 * `"any"` enforces always wrapping, but allows either style.
 
+Object option:
+
+* `"functionPrototypeMethods": true` enforces always wrapping `.call` and `.apply` uses consistently with the string option. The default is `false`
```

Comment:
I wonder if we can make this clearer - maybe something like `"functionPrototypeMethods": true additionally enforces wrapping function expressions invoked using .call and .apply.`?